exporter/exporterqueue/queue.go

```diff
@@ -24,7 +26,12 @@ type Queue[T any] queue.Queue[T]
 
 // Settings defines settings for creating a queue.
 type Settings struct {
-	DataType         component.DataType
+
+	// Deprecated: [v0.110.0] Use Signal instead
```

Comment:
Does this deprecated note work as expected if there's a nolint below it? i think alternatively nolint can be added to the end of line 32

Reply:
I'll move it to be safe